core/Hashing.h

```diff
@@ -32,7 +24,7 @@ inline unsigned int _hash(std::string_view utf8) {
         res = mix(res, *it - '!'); // "!" is the first printable letter in ASCII.
         // This will help Latin1 but may harm utf8 multibyte
     }
-    return res * HASH_MULT2 + _NameKind2Id_UTF8(NameKind::UTF8);
+    return res * HASH_MULT2 + static_cast<u4>(NameKind::UTF8);
```

Comment:
I don't know why we mix this in here, but this hash function is our hashing function for all strings in Sorbet.